Core/MIPS/MIPSIntVFPU.cpp

```diff
@@ -819,7 +819,7 @@ namespace MIPSInt
 			{
 				u32 value = s[0];
 				for (int i = 0; i < 4; i++) {
-					d[i] = (u32)((value & 0xFF) * 0x01010101) >> 1;
+					d[i] = (u32)((u32)(value & 0xFF) * 0x01010101UL) >> 1;
```

Comment:
Wow, did the & 0xFF produce a signed int output? But, the bottom 32 bits of a int*int should be the same as an uint*uint anyway, so I don't think this really mattered - though nice to have the types right?

Reply:
Err, oops.  I meant to undo this.  I thought something weird was going on, but it turned out to be `fabsf` modifying an integer that happened to also be a NAN.

-[Unknown]